src/schema.rs

```diff
@@ -328,6 +397,24 @@ impl Serial for ModuleV1 {
     }
 }
 
+impl Serial for VersionedModuleSchema {
+    fn serial<W: Write>(&self, out: &mut W) -> Result<(), W::Err> {
+        // Prefix for versioned module schema, used to distinquish from the unversioned.
+        out.write_u16(u16::MAX)?;
```

Comment:
This is a nitpick, but I would probably prefer if you were more explicit and wrote
```
out.write_all(&[255, 255])?
```

Reply:
But it is also fine the way it is now.

Reply:
I started out doing what you suggested, but later when deserializing a u16 seems a bit more convenient.